kotlin-reference-server/src/main/resources/default-config.yaml

```diff
@@ -12,6 +12,7 @@ app:
   distributionWallet: GBN4NNCDGJO4XW4KQU3CBIESUJWFVBUZPOKUZHT7W7WRB7CWOA7BXVQF
   distributionWalletMemo:
   distributionWalletMemoType:
+  secret: SAJW2O2NH5QMMVWYAN352OEXS2RUY675A2HPK5HEG2FRR2NXPYA4OLYN
```

Comment:
Does this require an account in the testnet? If yes, we need to add the key creation to the [useful-scirpts](https://github.com/stellar/useful-scripts/tree/main/anchor-platform-testnet-reset) repository.

Reply:
Yeah, I'll update the scripts.